Source/MSBuild.Sdk.Extras/Build/MSBuild.NET.Inference.targets

```diff
@@ -114,4 +106,10 @@
     <Import Condition="'$(TargetFrameworkVersion)' != ''" Project="$(MSBuildThisFileDirectory)FrameworkResolvers\MSBuild.NETPortable.v$(TargetFrameworkVersion).targets"/>
```

Comment:
It's so that this import should work. I could have done it a different way but this seems better in some sense, my reason is that any msbuild files within `FrameworkResolvers` folder can only have version number only, since any logic added afterwards involving setting `TFV` shouldn't worry about having `v` prefix, It just works at the end.

Reply:
Can it be    `<Import Condition="'$(TargetFrameworkVersion)' != ''" Project="$(MSBuildThisFileDirectory)FrameworkResolvers\MSBuild.NETPortable.$(TargetFrameworkVersion).targets"/> ` and leave the v in the variable? Is there a case where it'd only be the version?

Reassigning the variable like this may lead to subtle errors and make following the values in the trace harder.